crates/rust-analyzer/src/flycheck.rs

```diff
@@ -281,11 +281,14 @@ impl FlycheckActor {
                         }
                     }
 
-                    let command =
-                        match self.check_command(package.as_deref(), saved_file.as_deref()) {
-                            Some(c) => c,
-                            None => continue,
-                        };
+                    let command = match self.check_command(
```

Comment:
This can be rewritte as a `let else` statement now (there should be an assist for that)